WordPress/Classes/ViewRelated/Blog/Blog Dashboard/ViewModel/DashboardCardModel.swift

```diff
@@ -5,21 +5,33 @@ struct DashboardCardModel: Hashable {
     let id: DashboardCard
     let apiResponse: BlogDashboardRemoteEntity?
 
-    // Used as the `Hashable` to check if the cell should be updated or not
-    let apiResponseDictionary: NSDictionary?
+    /// Used as the `Hashable` to compare this `struct` to others
+    let hashableDictionary: NSDictionary?
 
-    init(id: DashboardCard, apiResponseDictionary: NSDictionary? = nil, entity: BlogDashboardRemoteEntity? = nil) {
+    /**
+     Initializes a new DashboardCardModel, used as a model for each dashboard card.
+
+     - Parameters:
+     - id: The `DashboardCard` id of this card
+     - hashableDictionary: A `NSDictionary?` that is used to compare this model to others
+     - entity: A `BlogDashboardRemoteEntity?` property
+
+     - Returns: A `DashboardCardModel` that is used by the dashboard diffable collection
+                view. The `hashableDictionary` and the `id` is used to differentiate one
+                card from the other.
+    */
+    init(id: DashboardCard, hashableDictionary: NSDictionary? = nil, entity: BlogDashboardRemoteEntity? = nil) {
         self.id = id
-        self.apiResponseDictionary = apiResponseDictionary
+        self.hashableDictionary = hashableDictionary
         self.apiResponse = entity
     }
 
     static func == (lhs: DashboardCardModel, rhs: DashboardCardModel) -> Bool {
         lhs.id == rhs.id &&
-        lhs.apiResponseDictionary == rhs.apiResponseDictionary
+        lhs.hashableDictionary == rhs.hashableDictionary
     }
 
     func hash(into hasher: inout Hasher) {
-        hasher.combine(apiResponseDictionary)
+        hasher.combine(hashableDictionary)
```

Comment:
Shouldn't we combine the `id` as well? 
It's mentioned in the comment that both `hashableDictionary` and `id` are used to identify a card. I realize that it may not be needed as of now, but I suggest adding it to be future-proof.

Reply:
Ops! Yes, we should. Thanks for catching this.

Done in 7a7d0232bf7d96ac458b178006762b50592f8c7d